app/controllers/download_controller.rb

```diff
@@ -26,6 +27,8 @@ def create
   def success; end
 
   def choose_access
+    @custom_dimension3 = @register.register_phase == 'Alpha' ? 'Alpha' : 'Live'
```

Comment:
we are repeating this logic multiple times, could we DRY it by moving it to a method on the model or something?

Reply:
Done 👍